### PR TITLE
Issue #SB-22286 fix: added configuration in header

### PR DESCRIPTION
**Pull request by Diptesh6501**

### CODE REVIEW

File: projects/sunbird-player-sdk/src/lib/player-utils/components/header/header.component.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChange
 export class HeaderComponent implements  OnChanges, OnInit {
   @Input() pageNumber;
   @Input() totalPages;
+  @Input() config;
```
Comment (HarishGangula) [inline projects/sunbird-player-sdk/src/lib/player-utils/components/header/header.component.ts:12]: Have Default values to show with setters and getters
it should be
```
{
  "rotation": {
    "clockWise": true,
    "anitClockWise": false
  },
  "goto": true,
  "navigation": true
}
```

Comment (HarishGangula) [reply projects/sunbird-player-sdk/src/lib/player-utils/components/header/header.component.ts:12]: use this and override from player same config add the type or interface for it

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: projects/sunbird-player-sdk/src/lib/player-utils/components/header/header.component.ts (hunk 1 above)
Comment (HarishGangula) [reply projects/sunbird-player-sdk/src/lib/player-utils/components/header/header.component.ts:12]: change code to below
```

private _config = {
    rotation:true,
    goto: true,
    navigation: true,
    zoom: true
  }
  
   get config(): any { 
    return this._config;
  }
  
  @Input()
  set config(value) {
    this._item = {..._config ,...value };
  }

```

**Review by HarishGangula — CHANGES_REQUESTED** [review]

**Comment by Diptesh6501** [comment]
raising the pull request to 3.7.0

**Pull request closed without merging**